module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -352,6 +352,8 @@ protected function getRssSearchResponse(ViewModel $view): Response
     protected function getSearchResultsView($setupCallback = null)
     {
         $view = $this->createViewModel();
+        $config = $this->getConfig('facets');
```

Comment:
You shouldn't hard-code facets.ini here, since the facet settings may reside in different files depending on the search backend. This should instead be `$config = $this->getConfig($this->getOptionsForClass()->getFacetsIni());`

If it makes sense to enable this functionality for other search backends (which it probably does), it may make sense to copy the new setting into all of the relevant .ini files -- EDS.ini, Summon.ini, etc., etc.

Reply:
Ok, changed
For the setting in other .ini files, most of them don't have the `Advanced_Settings` section, should I create it or should I move to another section?
In order to avoid several communications, can you confirm the list of config files concerned : 
(I'm not sure and implied all controllers extending AbstractSearch.php)
``` 
EDS.ini
EIT.ini
EPF.ini
Summon.ini
Primo.ini
Pazpar2.ini
LibGuidesAZ.ini
LibGuidesAPI.ini
LibGuides.ini
combined.ini
BrowZine.ini
Blender.ini
authority.ini
```

Reply:
Yes, I think it would make sense to move this to the [Results_Settings] section, since that's for settings specific to the search results (as opposed to the advanced search screen).

Reply:
The .ini files having the Results_Settings section are : 
- EDS.ini
- EPF.ini
- Summon.ini
- Blender.ini
The others don't have it
(I'm waiting on your answer to commit)

Reply:
The Results_Settings section is also present in facets.ini, website.ini, Search2.ini, Collection.ini and facets.ini (all of which are used for different types of Solr searches). Between the five Solr configurations and the four non-Solr configurations that you listed, that should cover all of the backends that support faceting, so I think it's okay to use this section to hold settings.

Reply:
I pushed the modifications made to : 
- EDS.ini
- EPF.ini
- Summon.ini
- Blender.ini
- Primo.ini
- facets.ini
- website.ini
- Search2.ini
- Collection.ini